autometa/common/external/prodigal.py

```diff
@@ -289,7 +289,8 @@ def main(args):
         format='%(asctime)s : %(name)s : %(levelname)s : %(message)s',
         datefmt='%m/%d/%Y %I:%M:%S %p',
         level=logger.DEBUG)
-    parser = argparse.ArgumentParser('Calls ORFs with provided input assembly')
+    parser = argparse.ArgumentParser(usage = 'prodigal.py' ,
```

Comment:
Adding the name of the script in usage is unnecessary as this is the default behavior.

Reply:
Yeah, but if we copy this code block to another `temp.py` and run `--help` then the usage would display `temp.py` and not `prodigal.py`

Reply:
This can also be automated in your `argparse_parser.py` script